content/docs/start/data-and-model-access.md

```diff
@@ -4,27 +4,27 @@ title: 'Get Started: Data and Model Access'
 
 # Get Started: Data and Model Access
 
-Okay, we've learned how to _track_ data and models with DVC, and how to commit
-their versions to Git. The next questions are: How can we _use_ these artifacts
-outside of the project? How do I download a model to deploy it? How to download
+We've learned how to _track_ data and models with DVC, and how to commit their
+versions to Git. The next questions are: How can we _use_ these artifacts
+outside of the project? How do we download a model to deploy it? How to download
 a specific version of a model? Or reuse datasets across different projects?
 
-> These questions tend to come up when you browse the files that DVC saves to
-> remote storage, e.g.
+> These questions tend to come up when you browse the files which DVC saves to
+> remote storage (e.g.
 > `s3://dvc-public/remote/get-started/fb/89904ef053f04d64eafcc3d70db673` 😱
-> instead of the original files, name such as `model.pkl` or `data.xml`.
+> instead of the original filename such as `model.pkl` or `data.xml`).
```

Comment:
For consistency (we have it separate in most docs):
```suggestion
> instead of the original file name such as `model.pkl` or `data.xml`).
```
> Committing this, please `git pull`.